hspec-core/hspec-core.cabal

```diff
@@ -28,6 +28,11 @@ source-repository head
   location: https://github.com/hspec/hspec
   subdir: hspec-core
 
+flag pretty-printing
+  default: True
+  manual: True
```

Comment:
Did you try if an "automatic" flag works instead?

Reply:
Indeed now I tried it and it works better, i.e. if the situation from #752 arises, Cabal disables the flag automatically.